colorizer_data/utils.py

```diff
@@ -245,6 +259,60 @@ def update_collection(
         json.dump(collection, f)
 
 
+def write_parquet_or_json_data(
```

Comment:
how about `write_data` ?  parquet or json is an implementation detail that a caller of this function doesn't need to care about, except for in the function parameters.

Reply:
Changed to `write_data_array`, since `write_data` is already a method in `ColorizerDatasetWriter`